publisher-sdk/src/main/java/com/criteo/publisher/Bid.java

```diff
@@ -24,7 +24,7 @@
 import com.criteo.publisher.util.AdUnitType;
 import kotlin.jvm.functions.Function1;
 
-public class BidResponse {
+public class Bid {
```

Comment:
Annotate with Keep and remove `BidResponse` from `proguard-rules`.

Reply:
`Bid#getPrice` is already `@Keep`. So no need to keep `Bid`. As this was done via annotation, there is no change in proguard file.

Reply:
What about `BidResponse`? We certainly don't need to keep in the proguard file since it doesn't exist anymore. I am talking about these rules:

```
-keep public class com.criteo.publisher.BidResponse
-keepclassmembers class com.criteo.publisher.BidResponse {
   public *;
}
```

Reply:
This is already removed in f462e55b which is now merged in `main` branch.